src/NHibernate/Persister/Entity/IEntityPersister.cs

```diff
@@ -623,5 +624,17 @@ public static int GetBatchSize(this IEntityPersister persister)
 
 			return 1;
 		}
+
+		internal static bool IsIdentifierMember(this IEntityPersister entityPersister, string memberPath)
```

Comment:
Copied from #2079. Added as an extension method so that can be used by other expression visitors.

Reply:
This method is now replaced by `EntityMetamodel.GetIdentifierPropertyType`.